corehq/motech/fhir/tests/test_api_endpoints.py

```diff
@@ -207,10 +209,6 @@ def get_foo_caseblock(owner_id):
             "display": "ZAF"
         }
     }],
-    # I don't know if `active` is required. If so, we can just use the
-    # template mentioned above to set `"active": True` for all cases. If
-    # it's not required, we should leave this out.
-    "active": True,
```

Comment:
What was the final thing on this now? It was not needed?

Reply:
It's definitely not an API requirement. If it's a requirement for a user of the API then creating a `FHIRResourceProperty` for it is simple.